src/pybind/mgr/mgr_module.py

```diff
@@ -1,10 +1,13 @@
+import sys
+
 import ceph_module  # noqa
 
-try:
-    from typing import Set, Tuple, Iterator, Any, Dict, Optional, Callable, List
-except ImportError:
-    # just for type checking
-    pass
+from typing import Set, Tuple, Iterator, Any, Dict, Optional, Callable, List, Union
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
```

Comment:
oddly, this is not needed:

https://github.com/python/typing/blob/master/typing_extensions/src_py3/typing_extensions.py#L543-L544